@here/harp-mapview/lib/ScreenCollisions.ts

```diff
@@ -72,7 +74,13 @@ export class ScreenCollisions {
      * @param bounds The bounding box in world coordinates.
      */
     allocate(bounds: Math2D.Box): void {
-        this.rtree.insert(bounds, null);
+        const bbox = {
+            minX: bounds.x,
+            minY: bounds.y,
+            maxX: bounds.x + bounds.w,
+            maxY: bounds.y + bounds.h
+        };
```

Comment:
it would be nice to remove the need of create bbox objects from Math2D.Box

Reply:
yes, I would also prefer to change the Math2D.Box layout to match the one from rbush.